### ISSUE TITLE
New copy for SweepBoard

### ISSUE THREAD

**Issue by quentinberder** [Wiredcraft/test-marketing#1]
We are pushing a new release of our product [SweepBoard](http://sweepboard.com/) after a lot of changes in the interfaces and some features added (you can find the full list in the log post). We revamped the landing page and added a new copy.

I would like you to review the copy of the [landing page](http://sweepboard.com/) while documenting and explaining your approach. You could also review the blog post announcing the latest release (coming up tonight, again document and review your approach). If you see any structural changes to be applied, or mistakes we should avoid in the page, outline them.

You can chat with us on our [HipChat channel](http://www.hipchat.com/gNE8HlDvc) and through GitHub issues.

**Comment by offbeat** [comment]
Perfect! I'll look forward to the blog post and rationalize/document any edits to that and the landing page.

**Comment by offbeat** [comment]
Revised blog copy below, along with screenshot of changes. I will post a followup with more detailed feedback and rationalization, but I thought you might want the copy more quickly in case you actually want to post it.

---

SweepBoard is getting sexier by the minute

We've just released a new version of SweepBoard, and we're pretty psyched! Not only did we clean up a handful of minor backlogged issues, we also added the much-anticipated multi-repositories support. We have additional features to roll out in the coming weeks, but let’s first recap what we've just pushed online:

• Multi repository support: you can now create boards that combine multiple GitHub repositories into a single Kanban board. This is particularly useful for management of large projects, which often span multiple repositories.
• Realtime updates from GitHub: changes on GitHub now immediately appear on SweepBoard. This means that your team can independently update and categorize issues with or without SweepBoard. Either way, your boards will always be synced and current.
• Wizard: we added a wizard for new users to walk them through the main features of SweepBoard.
• Pull requests: we are now styling pull requests differently from regular issues for emphasis and convenience.
• Github events: we are now displaying GitHub events (label changes, assignments, etc.) in the issues threads.
• We've had a bit of fun with our 404 page!

There are a lot more features coming in the next few weeks, including:

• Milestone support: allowing you to manage GitHub milestones from SweepBoard. This will integrate with the multi-repos support, allowing you to sync milestones across multiple repositories.
• Analytics: you will be able to access various metrics related to your projects, including activity, cycle time, contributors stats, and more.
• Optimized enterprise version: you'll be able to run SweepBoard on premises. Shoot us an email at info+enteprise@sweepboard.com if you're interested in becoming a SweepBoard beta tester.
• Many more changes and additions, ranging from minor tweaks to cool new features in the pipeline

Go to app.sweepboard.com to sign up: it's free.

Come chat with us if you have issues or suggestions. You can also drop us a line at info@sweepboard.com.

Happy kanban-ing!
![2014-09-25 06 34 44 pm](https://cloud.githubusercontent.com/assets/8875108/4403222/01a42664-44a0-11e4-8d60-88df2cf27cfb.png)

**Comment by quentinberder** [comment]
Will go over this tomorrow. Did you went through the copy of the landing page as well?

**Comment by offbeat** [comment]
SweepBoard is getting sexier by the minute – NOTES
General Philosophy
• Blog as “pull” strategy
     -  Keyword rich, easily indexed… essential to overall SEO strategy
     -  This strategy should be tethered to actionable goals (growth hacking essential), typically related to       acquisition of visitors and retention of members
     -  Compounding effect: more blogs = more chances to pull new visitors
     -  Education effect: people that are informed about the product are more likely to move through your funnel
     -  Thought Leader effect: see recommendation re: Kanban blog post to crosslink to. I’m not advocating producing fluff! Just mid- to high-level posts we can leverage for Thought Leader and SEO purposes
• SEO
     -  Content, content, content… blog posts become de facto landing pages via organic search traffic
     -  Optimized file names (see note re: image in post)
     -  SEO-optimized crosslinks: I identified four such opportunities in this blog post. REALLY important!
• Activating Members
     -  Basic strategies:
       •  Get their email address
       •  Get them to create an account
       •  Get them to read something
       •  Get them to fill out a form
       •  Get them to watch something
       •  Get them to interact with someone
     -  I’m all for a “soft sell,” but the blog posts need more calls to action (CTAs) to be strategically impactful beyond SEO. Growth is the one and only marketing imperative.
• Onboarding
     -  Provide context and orientation, lead visitors to where you want them to go
     -  Digital “tour guide” of sorts
     -  The “wizard” bullet point would be an excellent example if it linked to the actual wizard
     -  Love this quote: “People are not patient, and they don’t care about your product enough to find out how to use it. If you want to activate visitors, making them take certain actions, then you must carefully craft your onboarding experience.”

Specific Points re: Blog Edits
• Optimize file name for image: sweepboard-github-kanban-new-features.png or something similar would be my recommendation
• 1st paragraph:
     -  Minor verbiage changes: VERY fastidious about words and phrasing, so that’s not an indictment of the quality of previous text. I’m – frankly – obsessive about subtle nuances re: copy and the psychology of words and turns of phrase. Forgive me…
     -  For instance, “much-anticipated” is a subtle cue re: installed base
• 1st bullets:
     -  Same as above; no major surgery, just turns of phrase, punctuation stuff, and verbiage
     -  Wizard:
       •  Using “the main features of SweepBoard” as anchor text is great from an SEO standpoint
       •  And then crosslinking to the wizard itself is great for onboarding
     -  Pull Requests:
       •  Took a guess re: “emphasis and convenience” being the motivator here. If I’m wrong about the purpose, it would still be good to articulate the benefit to the user for each change mentioned
• 2nd bullets:
     -  Milestone support
       •  Using “manage GitHub milestones from SweepBoard” as anchor text is great from an SEO standpoint, provided we can quickly assemble a landing page on GitHub milestones that emphasizes the convenience of using SweepBoard to track/manage them. Said landing page should have a single, prominent CTA.
     -  Stable enterprise version
       •  Shy away from using words like this with a negative connotation when possible. I vote for “optimized,” but if the industry vocabulary requires “stable,” I have no objections. Good psychological best practice, but sometimes it’s unavoidable.
       •  Using “becoming a SweepBoard beta tester” as anchor text is great from SEO and onboarding standpoints, but it would require a dedicated landing page. Just something with a simple form and link to chat system.
     -  And much more
       •  My alternative is more verbose, but I try to use substantive teaser phrases where possible
• Other
     -  Concerned you guys might feel this is superficial and low-level, but it would be helpful from SEO and stickiness perspectives to do some one-off pages that talk about concepts in the context of your organizational experience and philosophy
     -  As an example, a “kanban” page would be helpful for crosslinks in blog posts, and industry professionals looking for a kanban solution would have higher odds of finding SweepBoard organically and converting into members

**Comment by offbeat** [comment]
Pivoting to landing page now... I'll email you the Word version of the above text because the formatting didn't transfer, and some of the bullet hierarchy is unclear.

**Comment by hunvreus** [comment]
I think it's important we keep in mind that we don't just need a new copy,
but also an actual strategy as yo what that content/layout should be and
what it should achieve.

Noah, you should hesitate to ask the team for a chat/info if you need to
clarify things about the product and features.

**Comment by offbeat** [comment]
@hunvreus absolutely! For the blog post, I just emailed you the 2-pages above about how I approach blog posts, using the SweepBoard one as an example. I think I addressed the content aspect, but yes: I may have more to contribute in that regard after some face time or chats with the team. Unfortunately, the soonest I can log some hours there is Monday. 

In regard to layout, I actually have zero qualms with the blog post as conceived. I'll have more on both topics for the landing page however. For the landing page especially, I'm giving myself a crash course in the most visceral pain points potential members experience. I will absolutely want to pick Wiredcraft brains though to confirm I have the main ones, and to catch any I may have missed.

**Comment by hunvreus** [comment]
I'd also check the stuff from Amy Hoy: http://unicornfree.com/ She has great tips in her blog for copy and marketing.

**Comment by hunvreus** [comment]
As well as the 30x500 newsletters (http://30x500.com).

**Comment by offbeat** [comment]
I've been on her site, I'm subscribed to the 30x500 newsletters (love the tone and content of their stuff by the way... good tip), I've been digesting the patio11 stuff you mentioned. Also, I had briefly worked with a developer who turned me on to developer.io, so I intuitively "get" the intercom.io opportunities (mentioned 2 in the screenshot comments relative to the blog). 

I think I get what you need from me: so let me take a less granular approach and speak more to overall strategy as well, both for the blog and landing page tasks. Is it problematic to get you revised blog notes and the landing page notes early next week in this context? Or do you want/need them sooner?

**Comment by offbeat** [comment]
@quentinberder are you in the office tomorrow? I can swing by to review both tasks with you in-person if you prefer. Monday is actually not looking so good for me on second glance, then the holiday hits, so tomorrow may be best on my end to get some face time with you and whomever else can give me info on SweepBoard features and the product itself. Then I could finish both tasks by this weekend at the latest.

**Comment by offbeat** [comment]
Hi guys, I got bogged down in urgent pre-holiday situations across the board, but I'm pivoting back to the SweepBoard stuff if you're still interested in evaluating me. @quentinberder I may have a few questions for you via chat if you'll be around this week to answer them. But I made some progress on my own, specifically getting a better sense of the ecosystem after ransacking waffle.io

@hunvreus I put pen to paper on the business development vs. sales vs. marketing conversation we briefly touched upon during our Skype, and I've spent more time with Amy Hoy, Alex Hillman, Patrick McKenzie, et al. So I'm in a better position to speak to that philosophy, which I will touch upon in my deliverables for the two tasks.

Any questions/comments/concerns, please reach out. My number one goal is advancing things with you guys, so please do not hesitate to advise me (beyond the guidance provided to-date, which has been greatly appreciated) to this end.

**Comment by quentinberder** [comment]
@offbeat we are back, you had some questions that needed answers?

**Comment by offbeat** [comment]
@quentinberder hey, good to know! Figured you had gone away for the holiday. I myself am back to the grind, putting out fires at the farm through Saturday. The Internet connection out here in Jiaxing is Third World, so I'll probably need to get into the chat feature this weekend or early next week. If you're around this weekend, a face-to-face may be more efficient, but I can consolidate the questions into a Github reply, no problem.

FYI I tapped the big guns for feedback though: my brother-in-law, Robert Kern - somewhat of a Python guru, and he gave me some good feedback on SweepBoard. 
https://github.com/rkern
https://www.enthought.com/company/team/devs/

So I'm close to where I need to be, but I was waiting to get back with you and/or Ronan to close some of the loops.
